src/modules/fancyzones/lib/Zone.cpp

```diff
@@ -84,12 +84,19 @@ void Zone::SizeWindowToZone(HWND window, HWND zoneWindow) noexcept
     ::GetWindowPlacement(window, &placement);
     placement.rcNormalPosition = zoneRect;
     placement.flags |= WPF_ASYNCWINDOWPLACEMENT;
+    // .shwoCmd will have either SW_SHOWNORMAL, SW_SHOWMINIMIZED or SW_SHOWMAXIMIZED
+    // depending on the window state. Keeping those values would make hidden windows
+    // visible again, so we will clear those flags, and set appropriate ones our
+    // selves.
```

Comment:
If a windows is hidden, shouldn't we skip it right away?

`ourselves` is one word.

Reply:
Fixed the word.

While generally windows get assigned the correct position when becoming visible, one specific window type - the New mail window broken by PowerToys without this commit - does not get zoned correctly without this.